src/components/OnboardingForm/OnboardingForm.tsx

```diff
@@ -182,6 +182,26 @@ export default function OnboardingForm() {
                         error={errors.major}
                     />
                 </div>
+                <div className="onboarding-form-waiver--column">
+                    <div className="onboarding-form-waiver--row">
+                        <h3 className="waiver-text">Please sign the following form:
+                            <a href="https://www.ams.ubc.ca/student-life/clubs/operating-a-club/club-constituency-general-membership-waiver/" className="waiver-link"> Insurance/Liability Waiver.</a>
+                        </h3>
+                    </div>
+
+                    <div className="waiver-check-row">
+
+                        <h3 className="waiver-text">I have signed the Insurance/Liability Waiver form.</h3>
+                        <input
```

Comment:
Small detail: style the cursor to pointer when you're hovering over the checkbox.